api/system/services/System.js

```diff
@@ -1,8 +1,106 @@
 'use strict';
+const fetch = require('node-fetch');
+
 
 /**
  * Read the documentation (https://strapi.io/documentation/3.0.0-beta.x/guides/services.html#core-services)
  * to customize this service
  */
 
-module.exports = {};
+module.exports = {
+  /**
+   * Promise to add record
+   *
+   * @return {Promise}
+   */
+
+  create: async (values) => {
+    let edsmData = await strapi.services.system.edsmUpdate(values);
```

Comment:
You should wrap your async/await with try/catch for catching errors. It's good practice. :)
So the code would look like:

`
try {
    let edsmData = await strapi.services.system.edsmUpdate(values);
    return strapi.query('System').create(edsmData);
} catch(error) {
    // your error handler
}
`